### PR TITLE
Upgrade k8s version to v1.21

### PR DESCRIPTION
**Pull request by prankulmahajan**
Need to first merge -> https://github.com/IBM/ibmcloud-volume-file-vpc/pull/9

### CODE REVIEW

File: pkg/ibmcsidriver/ibm_csi_driver.go [hunk 1]
```diff
@@ -24,11 +24,12 @@ import (
 
 	commonError "github.com/IBM/ibm-csi-common/pkg/messages"
 	nodeMetadata "github.com/IBM/ibm-csi-common/pkg/metadata"
+	mountManager "github.com/IBM/ibm-csi-common/pkg/mountmanager"
 	"github.com/IBM/ibm-csi-common/pkg/utils"
 	cloudProvider "github.com/IBM/ibmcloud-volume-file-vpc/ibmcloudprovider"
 	csi "github.com/container-storage-interface/spec/lib/go/csi"
 	"go.uber.org/zap"
-	"k8s.io/kubernetes/pkg/util/mount"
+	//mount "k8s.io/mount-utils"
```
Comment (sameshai) [inline pkg/ibmcsidriver/ibm_csi_driver.go:32]: Is this required ? can we remove commented code ?

Comment (prankulmahajan) [reply pkg/ibmcsidriver/ibm_csi_driver.go]: Removed

**Comment by prankulmahajan** [comment]
Successfully ran UT and sanity tests locally.

```
Ran 33 of 78 Specs in 0.110 seconds
SUCCESS! -- 33 Passed | 0 Failed | 1 Pending | 44 Skipped
--- PASS: TestSanity (0.12s)
PASS
ok  	github.com/IBM/ibm-vpc-file-csi-driver/tests/sanity	1.151s
```

**Review by sameshai — APPROVED** [review]
LGTM

**Review by meghnasiingh — APPROVED** [review]

**Pull request merged**